llvm/tools/llvm-mc/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ set(LLVM_LINK_COMPONENTS
   MC
   MCParser
   Support
+  DebugInfoDWARF
```

Comment:
The linker error is probably because we're adding the dependency in the `llvm-mc` tool, instead of the MC _library_: `lib/MC/CMakeLists.txt`

Reply:
Yep this is needed, hopefully it fixes the linux issue!